tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_rocm

```diff
@@ -16,6 +16,13 @@ LABEL maintainer="The ManyLinux project"
 
 RUN yum remove -y devtoolset\* git\* && conda remove -y cmake
 
+# remove protobuf 2.6.1 from rocm/pytorch:rocm4.2_centos7_py3.6_pytorch
+# it's too old to compile onnx 1.10
+RUN rm -fr /usr/local/bin/protoc \
```

Comment:
Then you need to delete them again before starting building onnx runtime. Otherwise it's hard to know which protobuf headers will be used: the ones in our source directory or the ones in /usr/local/include?

Reply:
OK, removed at end.

And passing at https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=496093&view=results